tests/parm/gocart/gefs/AERO_ExtData.rc

```diff
@@ -0,0 +1,152 @@
+# BROWN CARBON
+# ------------
+# Biomass burning -- QFED-v2.x
+BRC_BIOMASS NA  N Y %y4-%m2-%d2t12:00:00 none none biomass /dev/null
+
+# Terpene emission
+BRC_TERPENE NA  Y Y %y4-%m2-%d2t12:00:00 none none terpene /dev/null
+
+# Biofuel Source -- Included in AeroCom anthropogenic emissions
+BRC_BIOFUEL NA  Y Y %y4-%m2-%d2t12:00:00 none none biofuel /dev/null
+
+# Anthropogenic (BF & FF) emissions -- allowed to input as two layers
+BRC_ANTEBRC1 NA  Y Y %y4-%m2-%d2t12:00:00 none none anteoc1 /dev/null
+BRC_ANTEBRC2 NA  Y Y %y4-%m2-%d2t12:00:00 none none anteoc2 /dev/null
+
+# EDGAR based ship emissions
+BRC_SHIP    NA  Y Y %y4-%m2-%d2t12:00:00 none none oc_ship /dev/null
+
+# Aircraft fuel consumption
+BRC_AIRCRAFT NA  N Y %y4-%m2-%d2t12:00:00 none none none /dev/null
+
+# Aviation emissions during the three phases of flight
+BRC_AVIATION_LTO NA  Y Y %y4-%m2-%d2t12:00:00 none none oc_aviation /dev/null
+BRC_AVIATION_CDS NA  Y Y %y4-%m2-%d2t12:00:00 none none oc_aviation /dev/null
+BRC_AVIATION_CRS NA  Y Y %y4-%m2-%d2t12:00:00 none none oc_aviation /dev/null
+
+# SOA production
+pSOA_BIOB_VOC NA Y Y %y4-%m2-%d2t12:00:00 none none biofuel /dev/null
+
+# # ======= Nitrate Sources ========
+# EMI_NH3_AG   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_ag      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_EN   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_en      /dev/null
+# EMI_NH3_IN   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_in      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_RE   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_re      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_TR   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_tr      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_OC   'kg m-2 s-1'  Y Y      %y4-%m2-%d2T12:00:00    none     none     emiss_ocn   ExtData/PIESA/sfc/GEIA.emis_NH3.ocean.x576_y361.t12.20080715_12z.nc4
+
+# # --------------------------------------------------------------
+# # If using 64 levels please replace this section with the correct values (ie replace 127 with 64)
+# NITRATE_HNO3 'mol mol-1'   Y N      %y4-%m2-%d2T12:00:00    none     0.20     hno3        ExtData/PIESA/L127/GMI.vmr_HNO3.x144_y91.t12.2006.nc4
+# # --------------------------------------------------------------
+# NI_regionMask NA           Y V      -                       none     none     REGION_MASK ExtData/PIESA/sfc/ARCTAS.region_mask.x540_y361.2008.nc
+SU_BIOMASS NA  N Y %y4-%m2-%d2t12:00:00 none 0.7778 SO2 ExtData/QFED_Blended_20210325_50days.nc
+OC_BIOMASS NA  N Y %y4-%m2-%d2t12:00:00 none 0.7778 OC  ExtData/QFED_Blended_20210325_50days.nc
+BC_BIOMASS NA  N Y %y4-%m2-%d2t12:00:00 none 0.7778 BC  ExtData/QFED_Blended_20210325_50days.nc
```

Comment:
@aerorahul I believe these are the only custom files for the regression test, so as not to carry so much data on all platforms. ExtData is not staged yet...may be nice to be consistent between RT and workflow